src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -50,53 +52,65 @@
 import java.util.List;
 
 @Name("Entities")
-@Description("All entities in all worlds, in a specific world, in a chunk or in a radius around a certain location, " +
+@Description("All entities in all worlds, in a specific world, in a chunk, in a radius around a certain location or within two locations. " +
 		"e.g. <code>all players</code>, <code>all creepers in the player's world</code>, or <code>players in radius 100 of the player</code>.")
 @Examples({"kill all creepers in the player's world",
 		"send \"Psst!\" to all players within 100 meters of the player",
 		"give a diamond to all ops",
 		"heal all tamed wolves in radius 2000 around {town center}",
-		"delete all monsters in chunk at player"})
-@Since("1.2.1, 2.5 (chunks)")
+		"delete all monsters in chunk at player",
+		"size of all players within {_corner::1} and {_corner::2}}"})
+@Since("1.2.1, 2.5 (chunks), INSERT VERSION (within)")
 public class ExprEntities extends SimpleExpression<Entity> {
 
 	static {
 		Skript.registerExpression(ExprEntities.class, Entity.class, ExpressionType.PATTERN_MATCHES_EVERYTHING,
 				"[(all [[of] the]|the)] %*entitydatas% [(in|of) ([world[s]] %-worlds%|1¦%-chunks%)]",
 				"[(all [[of] the]|the)] entities of type[s] %entitydatas% [(in|of) ([world[s]] %-worlds%|1¦%-chunks%)]",
 				"[(all [[of] the]|the)] %*entitydatas% (within|[with]in radius) %number% [(block[s]|met(er|re)[s])] (of|around) %location%",
-				"[(all [[of] the]|the)] entities of type[s] %entitydatas% in radius %number% (of|around) %location%");
+				"[(all [[of] the]|the)] entities of type[s] %entitydatas% in radius %number% (of|around) %location%",
+				"[(all [[of] the]|the)] %*entitydatas% within %location% and %location%",
+				"[(all [[of] the]|the)] entities of type[s] %entitydatas% within %location% and %location%");
```

Comment:
How do you feel about `between` as an alternative to `within`?

Reply:
I don't think it should be an alias because then it conflicts with the idea that within is a cuboid and between is a line. I was thinking about adding between as a line but tbh it doesn't really feel like a useful addition and I feel like it would be a harder implementation anyway

Reply:
I agree with Phil. In Skript "within" has always referenced a cuboid and "between" references a straight line between two locations.